package com.senati.modaapp.data.model

data class Categoria(
    val id: Int = 0,
    val nombre: String
) {
    override fun toString(): String = nombre
}
